src/Common/src/System/Net/Mail/MailAddressParser.cs

```diff
@@ -30,6 +31,24 @@ internal static MailAddress ParseAddress(string data)
             return parsedAddress;
         }
 
+        // Parse a comma separated list of MailAddress's
+        //
+        // Throws a FormatException if any MailAddress is invalid.
+        internal static IList<MailAddress> ParseMultipleAddresses(string data)
```

Comment:
Nit: I realize the desktop code returns `IList<MailAddress>`, but it would be more efficient for callers to enumerate the result if this was changed to `List<MailAddress>`.